third_party/move/move-compiler-v2/tests/lambda/storable/registry.move

```diff
@@ -15,7 +15,7 @@ module 0x42::test {
 
     fun get_function(v: &vector<Function>, k: u64): Option<Function> {
         mut x = Option<Function>::None;
-        for_each_ref(v, |f: &Function| {
+        vector::for_each_ref(v, |f: &Function| {
```

Comment:
Is this added on purpose for better coverage on the parser code?

Reply:
I added this example to try to explore a potential use case.  I've been correcting various other errors as they show up, after being shadowed by unimplemented lambda errors.  I should probably get it closer to working, and fix all the unrelated errors.